nov-13-hardfork-spec.md

```diff
@@ -0,0 +1,38 @@
+# November 13th Bitcoin Cash Hardfork Technical Details
+
+Version 1.0, 2017-12-01
+
+## Summary
+ 
+When the median time past[1] of the most recent 11 blocks (MTP-11) is greater than or equal to UNIX timestamp 1510600000 Bitcoin Cash will execute a hardfork according to this specification.  It includes the following three protocol changes:
+
+* Enforcement of LOW_S signatures (BIP 0146)
+* Enforcement of NULLFAIL (BIP 0146)
+* A replacement for the emergency difficulty adjustment. The algorithm for the new difficulty adjustment is described below
+
+## Difficulty Adjustment Algorithm Description
+
+To calculate the difficulty of a given block (Bn+1), with an MTP-11[1] greater than or equal to the unix timestamp 1510600000, perform the following steps:
+
+1. Let Bn be the Nth block in a Bitcoin Cash Blockchain.
+1. Let Blast be the middle block of the [Bn, Bn-1, Bn-2] when sorted by timestamp.
+1. Let Bfirst be the middle block of the [Bn-144, Bn-145, Bn-146] when sorted by timestamp.
+1. Let the Timespan (TS) be equal to the difference in unix timestamps between Blast and Bfirst within the range [72 * 600, 288 * 600].  Values outside should be treated as their respective limits
+1. Let the Work Performed (W) be equal to the difference in chainwork between Blast  and Bfirst.
```

Comment:
For "chainwork" : 
 
Can we give a link to a C++ codebase function that returns this, so the computation is clearly defined, or, if simple enough to define it in-spec, put it in some kind of appendix and define it here.

I'm a bit worried that the definition of it is not clear to whoever reads the spec, and that even if it is, maybe it is implemented with slight deviations in other software, and could lead to problems.

This is also why this spec needs test vectors.